tests/functional/get_installed_test.py

```diff
@@ -48,13 +50,17 @@ def test_pip_get_installed(tmpdir):
         'git+git://github.com/bukzor/cov-core.git@master#egg=cov-core',
         '-e', 'git+git://github.com/bukzor/pytest-cov.git@master#egg=pytest-cov',
     )
+
     expected = [
-        'attrs', 'cov-core', 'coverage', 'more-itertools', 'pluggy', 'py',
-        'pytest', 'pytest-cov', 'six',
+        'atomicwrites', 'attrs', 'cov-core', 'coverage', 'more-itertools',
+        'pluggy', 'py', 'pytest', 'pytest-cov', 'six',
     ]
     if PY2:  # :pragma:nocover:
-        expected.insert(3, 'funcsigs')
-    assert get_installed() == expected
+        expected.extend(['funcsigs', 'scandir'])
+    if sys.version_info < (3, 6):
```

Comment:
probably just test python2.7 and python3.6 and add this to the python2 block.  python3.5 isn't giving us useful additional coverage

Reply:
According to the travis file we do run on python3.5 and pytest defines `pathlib2` for python<3.6
https://github.com/pytest-dev/pytest/blob/4d8903fd0b89369e0e44f084e93c43d2ce26d736/setup.py#L87

Reply:
yes, I am suggesting to stop that because there's no python3.5 specific code and yelp doesn't really use python3.5 all that much